shark-graph/src/test/java/shark/LongScatterSetTest.kt

```diff
@@ -1,142 +1,137 @@
 package shark
 
-import org.assertj.core.api.Assertions
 import org.junit.Test
 import shark.LongScatterSetAssertion.Companion.assertThat
-import shark.internal.hppc.HHPC
 import shark.internal.hppc.LongScatterSet
 
 class LongScatterSetTest {
 
-  @Test fun `verify LongScatterSet#add operation works correctly`() {
-    val set = LongScatterSet()
-    runAssertionsForAddOperation(set)
+  @Test fun `new set is empty`() {
+    assertThat(LongScatterSet()).isEmpty()
   }
 
-  @Test fun `verify LongScatterSet#remove operation works correctly`() {
+  @Test fun `adding elements`() {
     val set = LongScatterSet()
-    runAssertionsForRemoveOperation(set)
+
+    TEST_VALUES_LIST.forEach { set.add(it) }
+
+    assertThat(set)
+        .contains(*TEST_VALUES_LIST.toLongArray())
+        .hasSize(TEST_VALUES_LIST.size)
   }
 
-  @Test fun `verify LongScatterSet#release operation works correctly`() {
+  @Test fun `+= operator works as addition`() {
     val set = LongScatterSet()
-    set.add(42)
-    set.release()
+
+    set.add(TEST_VALUE)
```

Comment:
the name of the test says "+=" but the test calls add directly?

Reply:
also I'm not sure what value exactly we get from testing this?

Reply:
Verification that we didn't accidentally break the operator. It's trivial, but still..